package.json

```diff
@@ -1,27 +1,33 @@
 {
-  "name": "q-applet-server-ping",
+  "name": "daskeyboard-applet--server-ping",
   "displayName": "Server Ping",
-  "version": "1.0.0",

```

Comment:
Do you want users to rewalk through the configuration when upgrading or no ? 

If you want the user to reconfigure there applets when upgrading you need to increment the major version. E.g 2.0.0 
If not you need to make sure that the new index.js will work with the old `questions` keys

Reply:
I reviewed it and I don't think they do.  I kept all the original configuration elements intact and added one, but the applet is self-sustaining in terms of default configuration so it's fine if they don't configure the new `desiredStatusCode` param

Reply:
The status code used to be hard-coded to `200`.  Now it defaults to `200` but can also be changed if the user so desires